app/src/main/java/edu/uc/haflga/financialpal/Income.kt

```diff
@@ -4,7 +4,7 @@ import android.os.Bundle
 import androidx.appcompat.app.AppCompatActivity
 import edu.uc.haflga.financialpal.ui.income.IncomeFragment
 
-class income : AppCompatActivity() {
+class Income : AppCompatActivity() {
```

Comment:
Changing name from income -> Income to adhere to proper naming conventions of UpperCamelCase for classes

Reply:
Yes... this is good guidance.